internal-server.md

```diff
@@ -131,8 +131,7 @@ To download an existing repository use `git clone`:
 ```
 $ git clone git@data.neuro.polymtl.ca:datasets/data-single-subject
 $ cd data-single-subject
-$ git annex init
-$ git annex sync --content
+$ git annex get .
```

Comment:
minor comment about git-annex commands in general: i prefer using `git-annex` because it is faster to type (git-a + TAB completion).

Reply:
If you `brew install bash-completion` then it works both ways:

```
[kousu@requiem ~]$ git-a[TAB]
[kousu@requiem ~]$ git-annex
```

```
[kousu@requiem ~]$ git a[TAB]
add                             annex                           annex-remote-datalad-archives   annex-remote-osf                apply                           
am                              annex-remote-datalad            annex-remote-ora                annex-shell                     archive                         
[kousu@requiem ~]$ git an[TAB]nex
[kousu@requiem ~]$ git annex
```

and the benefit then is it will also complete git-annex commands:

```
[kousu@requiem ~]$ git annex a[TAB][TAB]
add        addunused  addurl     adjust     assistant  
[kousu@requiem ~]$ git annex g[TAB][TAB]
get          group        groupwanted  
```

Reply:
that is pretty cool-- i do suspect however that 99% of users like me were not aware of bash completion and so suggesting git-annex as usage will benefit both the 99% like me the 1% who do have bash-completion

Reply:
I really think we should be consistent. Everything in https://git-annex.branchable.com/walkthrough/ uses the `git annex`. The intent is that it's meant to be used as a `git` subcommand. The fact that it works as `git-annex` is a quirk, and not part of its API.

Reply:
unfortunately the brew instlal bash completion did not work for me:


<details><summary>Terminal output</summary>

```console
julien-macbook:~/data.neuro/basel-mp2rage $ brew install bash-completion
Error: 
  homebrew-core is a shallow clone.
  homebrew-cask is a shallow clone.
To `brew update`, first run:
  git -C /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core fetch --unshallow
  git -C /usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask fetch --unshallow
This restriction has been made on GitHub's request because updating shallow
clones is an extremely expensive operation due to the tree layout and traffic of
Homebrew/homebrew-core and Homebrew/homebrew-cask. We don't do this for you
automatically to avoid repeatedly performing an expensive unshallow operation in
CI systems (which should instead be fixed to not use shallow clones). Sorry for
the inconvenience!
==> Downloading https://homebrew.bintray.com/bottles/bash-completion-1.3_3.catalina.bottle.tar.gz
######################################################################## 100.0%
==> Pouring bash-completion-1.3_3.catalina.bottle.tar.gz
==> Caveats
Add the following line to your ~/.bash_profile:
  [[ -r "/usr/local/etc/profile.d/bash_completion.sh" ]] && . "/usr/local/etc/profile.d/bash_completion.sh"

Bash completion has been installed to:
  /usr/local/etc/bash_completion.d
==> Summary
🍺  /usr/local/Cellar/bash-completion/1.3_3: 189 files, 607.9KB
==> `brew cleanup` has not been run in 30 days, running now...
Removing: /Users/julien/Library/Caches/Homebrew/tree--1.8.0.catalina.bottle.tar.gz... (49.5KB)
Removing: /Users/julien/Library/Caches/Homebrew/ttyrec--1.0.8_1.catalina.bottle.tar.gz... (11.8KB)
Removing: /Users/julien/Library/Caches/Homebrew/Cask/monitorcontrol--2.1.0.dmg... (13.6MB)
Removing: /Users/julien/Library/Logs/Homebrew/libyaml... (64B)
Removing: /Users/julien/Library/Logs/Homebrew/ansible... (64B)
Removing: /Users/julien/Library/Logs/Homebrew/ttyrec... (64B)
Pruned 0 symbolic links and 1 directories from /usr/local
julien-macbook:~/data.neuro/basel-mp2rage $ 
julien-macbook:~/data.neuro/basel-mp2rage $ 
julien-macbook:~/data.neuro/basel-mp2rage $ git ann
julien-macbook:~/data.neuro/basel-mp2rage $ git a

```

</details>

i'll personally continue using git-annex

Reply:
You probably need to restart your shell, like we tell people to do with the SCT installer.

But if you're happy with `git-a[TAB]` then :+1: . I just don't want to confuse people by documenting it.

Reply:
> You probably need to restart your shell, like we tell people to do with the SCT installer.

It didn't help ☹. Anyway it's really not a big deal.